kitchen-tests/cookbooks/installthings/attributes/default.rb

```diff
@@ -0,0 +1,8 @@
+case node['platform_family']
+when "debian"
+  default['installthings']['push_client_url'] =
+    'https://opscode-private-chef.s3.amazonaws.com/ubuntu/12.04/x86_64/opscode-push-jobs-client_1.1.5-1_amd64.deb'
```

Comment:
that sounds easier

On Sat, Mar 14, 2015 at 4:02 PM Lamont Granquist <notifications@github.com>
wrote:

> In kitchen-tests/cookbooks/installthings/attributes/default.rb
> <https://github.com/chef/chef/pull/3076#discussion_r26442706>:
>
> > @@ -0,0 +1,8 @@
> > +case node['platform_family']
> > +when "debian"
> > +  default['installthings']['push_client_url'] =
> > +    'https://opscode-private-chef.s3.amazonaws.com/ubuntu/12.04/x86_64/opscode-push-jobs-client_1.1.5-1_amd64.deb'
>
> like
> https://github.com/chef/chef/blob/master/spec/data/apt/chef-integration-test_1.0-1_amd64.deb
>
> could use fpm to create an equivalently trivial rpm to install.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chef/chef/pull/3076/files#r26442706>.
>